lib/util/http/http.ts

```diff
@@ -176,11 +172,26 @@ export abstract class HttpBase<
           )
         : null;
 
+    if (memCacheKey) {
+      cacheProvider = undefined;
+    }
+
+    const cachedResponse = await cacheProvider?.bypassServer<unknown>(url);
+    if (cachedResponse) {
+      return cachedResponse;
+    }
+
     let resPromise: Promise<HttpResponse<unknown>> | null = null;
 
     // Cache GET requests unless memCache=false
     if (memCacheKey) {
       resPromise = memCache.get(memCacheKey);
+
+      /* v8 ignore start: temporary code */
+      if (resPromise && options.cacheProvider !== memCacheProvider) {
+        logger.debug({ url }, 'Cache hit on the obsolete memCache option');
```

Comment:
Add these URLs to a set, and then log the full list at the end of a run (instead of one debug at a time)